addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/protocol/KodiConnection.java

```diff
@@ -211,22 +208,20 @@ private void updateFanartUrl(String imagePath) {
         /*
```

Comment:
Maybe git blame already us with that?
@pail23 do you have any memories about it?